src/pods/canvas/canvas.pod.tsx

```diff
@@ -93,6 +94,9 @@ export const CanvasPod: React.FC = () => {
           );
         </svg>
       </div>
+      <ModalDialog isOpen={true}>
```

Comment:
Mmm... let's simulate the whole cycle:
  - Create a button on the toolbar to e.g. create a relation (see mocks in wiki).
  - When you click in the button a modal dialog is opened

Create a second simulation: click on edit button should open another modal to edit the table (just place a text inside)

Reply:
It's fixed, but to improve the use of the modal, I have created a provider. The modal is only present in the initial scene. Then we use "useModalDialogContext" to open and close it, modifying the component it hosts. Currently, it's used exclusively to render components inside the modal. It may need a nomenclature change.